pkg/nfd-worker/nfd-worker.go

```diff
@@ -762,10 +765,19 @@ func (m *nfdWorker) updateNodeFeatureObject(labels Labels) error {
 		nfrUpdated.Labels = map[string]string{nfdv1alpha1.NodeFeatureObjNodeNameLabel: nodename}
 		nfrUpdated.OwnerReferences = ownerRefs
 		nfrUpdated.Spec = nfdv1alpha1.NodeFeatureSpec{
-			Features: *features,
+ 			Features: *features,
 			Labels:   labels,
 		}
 
+		currentTime := time.Now()
+		updated := fmt.Sprintf("Updated: %q", currentTime)
+		nFeatures := fmt.Sprintf("Number of feature sources: %q", len(m.featureSources))
+		nFeatureErrors := fmt.Sprintf("Number of sources that had failuers: %q", m.featureSourceErrors)
+		nLabels := fmt.Sprintf("Number of label sources: %q", len(m.labelSources))
+
+		nfrUpdated.Status = nfdv1alpha1.NodeFeatureStatus{
+			Status: []string{ updated, nFeatures, nFeatureErrors, nLabels },
+		}
```

Comment:
I think we should put status details each in a different field and not in the same string. What I mean here is that we would have the following:
```
status:
   updatedAt: xxxx
   numberOfFeatures: xxxx
   numberOfLabels: xxxx
```
This is how it's usually done. wdyt @ozhuraki 
Thoughts on this @marquiz @ArangoGutierrez

Reply:
@TessaIO
> I think we should put status details each in a different field and not in the same string

Could be done this way too, let's wait for more input.

Reply:
If I get your implementation correctly, the status in the Kubernetes object would be like this:
```
status: 123456789 2 3 5
```
It's really hard for the user to interpret this, that's why I suggested that we put them in separate, yet descriptive fields.

Reply:
I agree that having a list of strings in an API is not very user friendly, how someone will refer to those fields in programmatic way? Will we sort the conditions etc.
It should be concrete fields with appropriate types, like integer for failure numbers etc.